### PR TITLE
VACMS-3208: ensure that created terms are cleaned up after test is complete

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #3208 

## Testing done

Ran behat test locally and verified that terms are created and cleaned up.

### CODE REVIEW

File: tests/behat/features/content_editing.feature [hunk 1]
```diff
@@ -24,25 +24,19 @@ Feature: CMS Users may effectively create & edit content
 
 @content_editing
   Scenario: Log in and confirm that "Checklist" node edit has the correct field settings
-    When I am logged in as a user with the "administrator" role
+    When I am logged in as a user with the "content_admin" role
```
Comment (acrollet) [inline tests/behat/features/content_editing.feature:27]: @ndouglas sauce for the goose!

File: tests/behat/features/content_editing.feature [hunk 2]
```diff
@@ -24,25 +24,19 @@ Feature: CMS Users may effectively create & edit content
...
+
+    Given "topics" terms:
+    | name            | parent | description | format     | language |
+    | BeHat - Topic 1 |        |             | plain_text | UND      |
+    | BeHat - Topic 2 |        |             | plain_text | UND      |
+    | BeHat - Topic 3 |        |             | plain_text | UND      |
+    | BeHat - Topic 4 |        |             | plain_text | UND      |
```
Comment (swirtSJW) [inline tests/behat/features/content_editing.feature:34]: That is a lot cleaner.    Interesting that this causes them to be cleaned up naturally.

File: tests/behat/features/content_editing.feature [hunk 3]
```diff
@@ -24,25 +24,19 @@ Feature: CMS Users may effectively create & edit content
 
 @content_editing
   Scenario: Log in and confirm that "Checklist" node edit has the correct field settings
-    When I am logged in as a user with the "administrator" role
```
Comment (swirtSJW) [inline tests/behat/features/content_editing.feature:27]: Is this change related to the issue, or just an  improvement?

File: tests/behat/features/content_editing.feature (hunk 2 above)
Comment (acrollet) [reply tests/behat/features/content_editing.feature:34]: definitely! I believe it's because it comes from the Drupal extension and it handles the cleanup.

File: tests/behat/features/content_editing.feature (hunk 3 above)
Comment (acrollet) [reply tests/behat/features/content_editing.feature:27]: just an improvement that I thought of when reviewing @ndouglas's PR.

File: tests/behat/features/content_editing.feature (hunk 2 above)
Comment (ElijahLynn) [reply tests/behat/features/content_editing.feature:34]: Thanks, learned something!

**Review by swirtSJW — APPROVED** [review]
